verinfast/agent.py

```diff
@@ -118,11 +121,13 @@ def main():
             # Check if SEMGrep is installed
             checkDependency("semgrep", "SEMGrep")
 
+            # Check if SEMGrep is installed
```

Comment:
Should update comment

Reply:
I updated the comment. I also noticed we were doing "Check Dependency" for AWS and Azure, but not "gcloud", so I added that and added the output of our Check Dependency check "which" to the debug log.

No hurry on reviewing any PRs